airflow/cli/commands/dag_command.py

```diff
@@ -441,3 +442,10 @@ def dag_test(args, session=None):
             _display_dot_via_imgcat(dot_graph)
         if show_dagrun:
             print(dot_graph.source)
+
+
+@provide_session
+@cli_utils.action_logging
+def dag_reserialize(args, session=None):
+    session.query(SerializedDagModel).delete()
+    session.commit()
```

Comment:
```suggestion
@provide_session
@cli_utils.action_logging
def dag_reserialize(args, session=None):
    session.query(SerializedDagModel).delete(synchronize_session=False)
```

Skip synchronisation to make the operation slightly faster. The session is going to end immediately after this so there’s no point to keep things in sync.

The `provide_session` decorator calls `commit()` automatically, we don’t need to do that again.